src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -331,6 +353,12 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
         # Change --timeout format for helm client to understand
         onboarding_timeout = onboarding_timeout + "s"
         cmd_helm_install.extend(["--wait", "--timeout", "{}".format(onboarding_timeout)])
+    if container_registry_repository:
+        cmd_helm_install.extend(["--set", "systemDefaultValues.image.repository={}".format(container_registry_repository)])
+        if container_registry_username and container_registry_password:
+            cmd_helm_install.extend(["--set", "systemDefaultValues.image.username={}".format(container_registry_username)])
```

Comment:
Introducing these new params in the helm charts would definitely mean that the agents compatible with PCR feature will not have the current version (1.3.8),  but a greater version. This means we cannot not use 1.3.8 hardcoding in the registry_path

Reply:
added a new parameter to take the agent version from user